apps/core/templatetags/core_tags.py

```diff
@@ -20,3 +20,8 @@ def navigation(context):
         "current_page": context.get("page"),
         "annotated_list": Page.get_annotated_list_qs(pages),
     }
+
+
+@register.filter
+def hreflang_url(value, arg):
+    return value.replace(arg, "en-latest")
```

Comment:
Would this cause problems if the page's path has its own `language_code` as a substring? e.g. for a page with the slug `upgrade-latest-version` under the Deutsch latest (`de-latest`) locale, this tag will return something like `/en-latest/upgraen-latest-version`.

It's somewhat of an edge case, but maybe there's something we can do to prevent this (e.g. maybe replacing `/{arg}/` with `/en-latest/` instead of just `arg` with `en-latest`, or alternatively we could also get the `full_url` of the en-latest version of the page).

Reply:
Nice catch @laymonage! I think we can just specify `count` parameter in `replace`. For eg `value.replace(arg, "en-latest", 1)`. This will replace only the first instance of the given string. And `/de-latest/` will always appear before any other instance like `upgrade-latest-version` slug like you mentioned above.

Reply:
Yep, I think that works too!